test_jsonschema.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2022 Axis Communications AB.
```

Comment:
Can't recall what we usually do with the years in the copyright notices, but:

```suggestion
# Copyright 2022-2023 Axis Communications AB and others.
```

Reply:
In this case it is quite obvious that some of the code is taken from an existing file copyrighted to Axis, so I believe the copyright should include Axis even though this is a new file committed by a non-Axis employee.